app/static/tests/e2e/code.etest.ts

```diff
@@ -53,12 +58,15 @@ const expectMonacoDecoration = async (state: any, line: number, numOfLines: numb
         } else if (i === numOfLines - 1) {
             expect(lineElement).toHaveClass("current-line");
             expect(glyphElement.nth(0)).toHaveClass("current-line current-line-margin-both");
-            expect(glyphElement.nth(1)).toHaveClass("active-line-number line-numbers lh-odd");
+            // regex to check all three classes are in string, this was required as monaco sometimes switches the
+            // order of the classes on this element
+            expect(glyphElement.nth(1))
```

Comment:
Might be more readable to just put those on separate lines, but I know you love a regex! :laughing:

Reply:
ahh yes, I can do multiple lines! I will definitely try and sneak in regex somewhere in all the future work I do though :smiling_imp:

Reply:
Thanks!